diff --git mesonbuild/compilers/compilers.py mesonbuild/compilers/compilers.py
index 25835a3..89bfca0 100644
--- mesonbuild/compilers/compilers.py
+++ mesonbuild/compilers/compilers.py
@@ -992,12 +992,12 @@ class Compiler:
         if not rpath_paths and not build_rpath:
             return []
         # On OSX, rpaths must be absolute.
-        abs_rpaths = [os.path.join(build_dir, p) for p in rpath_paths]
-        if build_rpath != '':
-            abs_rpaths.append(build_rpath)
+        #KEN abs_rpaths = [os.path.join(build_dir, p) for p in rpath_paths]
+        #KEN if build_rpath != '':
+        #KEN     abs_rpaths.append(build_rpath)
         # Ensure that there is enough space for large RPATHs
         args = ['-Wl,-headerpad_max_install_names']
-        args += ['-Wl,-rpath,' + rp for rp in abs_rpaths]
+        #KEN args += ['-Wl,-rpath,' + rp for rp in abs_rpaths]
         return args
 
     def build_unix_rpath_args(self, build_dir, from_dir, rpath_paths, build_rpath, install_rpath):
@@ -1033,7 +1033,7 @@ class Compiler:
             # $ORIGIN in the runtime path will be undefined and any binaries
             # linked against local libraries will fail to resolve them.
             args.append('-Wl,-z,origin')
-        args.append('-Wl,-rpath,' + paths)
+        #KEN args.append('-Wl,-rpath,' + paths)
         if get_compiler_is_linuxlike(self):
             # Rpaths to use while linking must be absolute. These are not
             # written to the binary. Needed only with GNU ld:
diff --git mesonbuild/modules/gnome.py mesonbuild/modules/gnome.py
index 43cdb71..413743e 100644
--- mesonbuild/modules/gnome.py
+++ mesonbuild/modules/gnome.py
@@ -302,10 +302,10 @@ class GnomeModule(ExtensionModule):
             for d in state.backend.determine_rpath_dirs(lib):
                 d = os.path.join(state.environment.get_build_dir(), d)
                 link_command.append('-L' + d)
-                if include_rpath:
-                    link_command.append('-Wl,-rpath,' + d)
-            if include_rpath:
-                link_command.append('-Wl,-rpath,' + libdir)
+                #KEN if include_rpath:
+                #KEN     link_command.append('-Wl,-rpath,' + d)
+            #KEN if include_rpath:
+            #KEN     link_command.append('-Wl,-rpath,' + libdir)
             depends.append(lib)
         if gir_has_extra_lib_arg(self.interpreter) and use_gir_args:
             link_command.append('--extra-library=' + lib.name)
@@ -355,8 +355,8 @@ class GnomeModule(ExtensionModule):
                             getattr(dep, 'is_libtool', False)):
                         lib_dir = os.path.dirname(lib)
                         external_ldflags.update(["-L%s" % lib_dir])
-                        if include_rpath:
-                            external_ldflags.update(['-Wl,-rpath {}'.format(lib_dir)])
+                        #KEN if include_rpath:
+                        #KEN     external_ldflags.update(['-Wl,-rpath {}'.format(lib_dir)])
                         libname = os.path.basename(lib)
                         if libname.startswith("lib"):
                             libname = libname[3:]
