--- ./mesonbuild/linkers.py.orig	2020-03-26 16:17:42.000000000 -0700
+++ ./mesonbuild/linkers.py	2020-03-26 16:21:54.000000000 -0700
@@ -617,7 +617,7 @@
                         is_shared_module: bool) -> T.List[str]:
         if is_shared_module:
             return []
-        install_name = ['@rpath/', prefix, shlib_name]
+        install_name = ['@loader_path/', prefix, shlib_name]
         if soversion is not None:
             install_name.append('.' + soversion)
         install_name.append('.dylib')
@@ -645,7 +645,7 @@
         for rp in all_paths:
             args.extend(self._apply_prefix('-rpath,' + rp))
 
-        return args
+        return []
 
 
 class GnuDynamicLinker(GnuLikeDynamicLinkerMixin, PosixDynamicLinkerMixin, DynamicLinker):
