--- Portfile.orig	2016-12-29 08:51:40.000000000 -0800
+++ Portfile	2016-12-29 08:55:50.000000000 -0800
@@ -116,6 +116,14 @@
     }
 }
 
+
+platform darwin 8 {
+    configure.args-delete \
+                      --system-libs \
+                      --no-system-jsoncpp
+    depends_lib-delete path:lib/pkgconfig/libuv.pc:libuv
+}
+
 configure.args-append --docdir=share/doc/cmake \
                     --parallel=${build.jobs} \
                     --init=${worksrcpath}/macports.cmake \
