--- texk/web2c/Makefile.am.orig	2018-04-25 21:28:17.000000000 -0700
+++ texk/web2c/Makefile.am	2018-04-25 21:28:55.000000000 -0700
@@ -90,7 +90,7 @@
 ## Rebuild liblua52
 @LUA52_RULE@
 ## Rebuild libluajit
-@LUAJIT_RULE@
+## @LUAJIT_RULE@
 ## Rebuild libpng
 @LIBPNG_RULE@
 ## Rebuild libfreetype
--- texk/web2c/Makefile.in.orig    2018-04-26 15:50:23.000000000 -0700
+++ texk/web2c/Makefile.in 2018-04-26 15:50:32.000000000 -0700
@@ -14096,7 +14096,7 @@
 @PTEXENC_RULE@
 @ZLIB_RULE@
 @LUA52_RULE@
-@LUAJIT_RULE@
+##@LUAJIT_RULE@
 @LIBPNG_RULE@
 @FREETYPE2_RULE@
 @PIXMAN_RULE@
