diff -ur pingus-06afdabb6b27c1314bb1d90a05d467697b72be58.orig/external/tinygettext/src/iconv.cpp pingus-06afdabb6b27c1314bb1d90a05d467697b72be58/external/tinygettext/src/iconv.cpp
--- pingus-06afdabb6b27c1314bb1d90a05d467697b72be58.orig/external/tinygettext/src/iconv.cpp	2017-07-06 19:26:39.000000000 -0700
+++ pingus-06afdabb6b27c1314bb1d90a05d467697b72be58/external/tinygettext/src/iconv.cpp	2017-07-06 19:57:17.000000000 -0700
@@ -110,7 +110,7 @@
 
     // We try to avoid to much copying around, so we write directly into
     // a std::string
-    tinygettext_ICONV_CONST char* inbuf = const_cast<char*>(&text[0]);
+    tinygettext_ICONV_CONST const char* inbuf = const_cast<char*>(&text[0]);
     std::string result(outbytesleft, 'X');
     char* outbuf = &result[0];
 
diff -ur pingus-06afdabb6b27c1314bb1d90a05d467697b72be58.orig/src/engine/system/sdl_system.cpp pingus-06afdabb6b27c1314bb1d90a05d467697b72be58/src/engine/system/sdl_system.cpp
--- pingus-06afdabb6b27c1314bb1d90a05d467697b72be58.orig/src/engine/system/sdl_system.cpp	2017-07-06 19:26:39.000000000 -0700
+++ pingus-06afdabb6b27c1314bb1d90a05d467697b72be58/src/engine/system/sdl_system.cpp	2017-07-06 19:27:11.000000000 -0700
@@ -25,7 +25,7 @@
 
 SDLSystem::SDLSystem()
 {
-  if (SDL_Init(SDL_INIT_VIDEO | SDL_INIT_JOYSTICK) != 0)
+  if (SDL_Init(SDL_INIT_VIDEO) != 0)
   {
     log_error("Unable to initialize SDL: %1%", SDL_GetError());
     exit(1);
diff -ur pingus-06afdabb6b27c1314bb1d90a05d467697b72be58.orig/src/util/system.cpp pingus-06afdabb6b27c1314bb1d90a05d467697b72be58/src/util/system.cpp
--- pingus-06afdabb6b27c1314bb1d90a05d467697b72be58.orig/src/util/system.cpp	2017-07-06 19:26:39.000000000 -0700
+++ pingus-06afdabb6b27c1314bb1d90a05d467697b72be58/src/util/system.cpp	2017-07-06 19:59:14.000000000 -0700
@@ -476,23 +476,7 @@
 
   if (lang.empty() || lang == "C")
   {
-#ifndef __APPLE__
     return globals::default_language;
-#else /* on Mac OS X we get "C" if launched using Finder, so we ask the OS for the language */
-    /* Note: this is used as last resort to allow the use of LANG when starting via Terminal */
-    CFArrayRef preferred_languages = CFLocaleCopyPreferredLanguages();
-    //CFShow(preferred_languages);
-    CFStringRef language = (CFStringRef)CFArrayGetValueAtIndex(preferred_languages, 0 /* most important language */);
-    //CFShow(language);
-    CFRelease(preferred_languages);
-    CFStringRef substring = CFStringCreateWithSubstring(NULL, language, CFRangeMake(0, 2));
-    CFRelease(language);
-    char buff[3];
-    CFStringGetCString(substring, buff, 3, kCFStringEncodingUTF8);
-    CFRelease(substring);
-    lang = buff;
-    return lang;
-#endif
   }
   else
   {
