--- a/projects/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake.orig 2020-03-21 17:34:47.000000000 -0700
+++ b/projects/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake      2020-03-21 17:41:37.000000000 -0700
@@ -4,7 +4,7 @@
 # set the default Xcode to use. This function finds the SDKs that are present in
 # the current Xcode.
 function(find_darwin_sdk_dir var sdk_name)
-  set(DARWIN_${sdk_name}_CACHED_SYSROOT "" CACHE STRING "Darwin SDK path for SDK ${sdk_name}.")
+  set(DARWIN_${sdk_name}_CACHED_SYSROOT "/Developer/SDKs/MacOSX10.4u.sdk" CACHE STRING "Darwin SDK path for SDK ${sdk_name}.")
   set(DARWIN_PREFER_PUBLIC_SDK OFF CACHE BOOL "Prefer Darwin public SDK, even when an internal SDK is present.")
 
   if(DARWIN_${sdk_name}_CACHED_SYSROOT)
