--- js/src/jsgc.cpp.orig	2018-09-11 22:06:58.000000000 -0700
+++ js/src/jsgc.cpp	2018-09-11 22:09:02.000000000 -0700
@@ -103,6 +103,12 @@
 using namespace js;
 using namespace js::gc;
 
+/* for GetCPUCount() */
+//#include <stdio.h>
+#include <sys/param.h>
+#include <sys/sysctl.h>
+
+
 namespace js {
 namespace gc {
 
@@ -2865,8 +2871,19 @@
         GetSystemInfo(&sysinfo);
         ncpus = unsigned(sysinfo.dwNumberOfProcessors);
 # else
+/*
         long n = sysconf(_SC_NPROCESSORS_ONLN);
         ncpus = (n > 0) ? unsigned(n) : 1;
+*/
+        int mib[2];
+        unsigned int maxproc = 1;
+        size_t len = sizeof(maxproc);
+
+        mib[0] = CTL_HW;
+        mib[1] = HW_NCPU;
+        if (sysctl(mib, 2, &maxproc, &len, NULL, 0) == -1)
+                maxproc = 1;
+        ncpus = unsigned(maxproc);
 # endif
     }
     return ncpus;
