--- src/ld/Options.cpp.orig2	2019-01-29 21:27:48.000000000 -0800
+++ src/ld/Options.cpp	2019-01-29 21:28:43.000000000 -0800
@@ -3503,8 +3503,8 @@
 
 	// can't use -rpath unless targeting 10.5 or later
 	if ( fRPaths.size() > 0 ) {
-		if ( !minOS(ObjectFile::ReaderOptions::k10_5, ObjectFile::ReaderOptions::k2_0) )
-			throw "-rpath can only be used when targeting Mac OS X 10.5 or later";
+//KEN		if ( !minOS(ObjectFile::ReaderOptions::k10_5, ObjectFile::ReaderOptions::k2_0) )
+//KEN			throw "-rpath can only be used when targeting Mac OS X 10.5 or later";
 		switch ( fOutputKind ) {
 			case Options::kDynamicExecutable:
 			case Options::kDynamicLibrary:
